[2023-09-13T07:38:43.036+0000] {taskinstance.py:1103} INFO - Dependencies all met for dep_context=non-requeueable deps ti=<TaskInstance: weather_data.extract_data scheduled__2023-08-31T00:00:00+00:00 [queued]>
[2023-09-13T07:38:43.048+0000] {taskinstance.py:1103} INFO - Dependencies all met for dep_context=requeueable deps ti=<TaskInstance: weather_data.extract_data scheduled__2023-08-31T00:00:00+00:00 [queued]>
[2023-09-13T07:38:43.049+0000] {taskinstance.py:1308} INFO - Starting attempt 1 of 1
[2023-09-13T07:38:43.086+0000] {taskinstance.py:1327} INFO - Executing <Task(PythonOperator): extract_data> on 2023-08-31 00:00:00+00:00
[2023-09-13T07:38:43.097+0000] {standard_task_runner.py:57} INFO - Started process 126 to run task
[2023-09-13T07:38:43.144+0000] {standard_task_runner.py:84} INFO - Running: ['***', 'tasks', 'run', 'weather_data', 'extract_data', 'scheduled__2023-08-31T00:00:00+00:00', '--job-id', '30', '--raw', '--subdir', 'DAGS_FOLDER/weather_dag.py', '--cfg-path', '/tmp/tmprtfw3o3k']
[2023-09-13T07:38:43.162+0000] {standard_task_runner.py:85} INFO - Job 30: Subtask extract_data
[2023-09-13T07:38:43.286+0000] {task_command.py:410} INFO - Running <TaskInstance: weather_data.extract_data scheduled__2023-08-31T00:00:00+00:00 [running]> on host 190733975a8f
[2023-09-13T07:38:43.531+0000] {taskinstance.py:1547} INFO - Exporting env vars: AIRFLOW_CTX_DAG_OWNER='***' AIRFLOW_CTX_DAG_ID='weather_data' AIRFLOW_CTX_TASK_ID='extract_data' AIRFLOW_CTX_EXECUTION_DATE='2023-08-31T00:00:00+00:00' AIRFLOW_CTX_TRY_NUMBER='1' AIRFLOW_CTX_DAG_RUN_ID='scheduled__2023-08-31T00:00:00+00:00'
[2023-09-13T07:38:43.865+0000] {python.py:183} INFO - Done. Returned value was: None
[2023-09-13T07:38:43.891+0000] {taskinstance.py:1350} INFO - Marking task as SUCCESS. dag_id=weather_data, task_id=extract_data, execution_date=20230831T000000, start_date=20230913T073843, end_date=20230913T073843
[2023-09-13T07:38:43.965+0000] {local_task_job_runner.py:225} INFO - Task exited with return code 0
[2023-09-13T07:38:43.989+0000] {taskinstance.py:2653} INFO - 0 downstream tasks scheduled from follow-on schedule check
[2023-09-15T11:31:18.416+0000] {taskinstance.py:1103} INFO - Dependencies all met for dep_context=non-requeueable deps ti=<TaskInstance: weather_data.extract_data scheduled__2023-08-31T00:00:00+00:00 [queued]>
[2023-09-15T11:31:18.435+0000] {taskinstance.py:1103} INFO - Dependencies all met for dep_context=requeueable deps ti=<TaskInstance: weather_data.extract_data scheduled__2023-08-31T00:00:00+00:00 [queued]>
[2023-09-15T11:31:18.437+0000] {taskinstance.py:1308} INFO - Starting attempt 1 of 1
[2023-09-15T11:31:18.471+0000] {taskinstance.py:1327} INFO - Executing <Task(PythonOperator): extract_data> on 2023-08-31 00:00:00+00:00
[2023-09-15T11:31:18.494+0000] {standard_task_runner.py:57} INFO - Started process 361 to run task
[2023-09-15T11:31:18.511+0000] {standard_task_runner.py:84} INFO - Running: ['***', 'tasks', 'run', 'weather_data', 'extract_data', 'scheduled__2023-08-31T00:00:00+00:00', '--job-id', '21', '--raw', '--subdir', 'DAGS_FOLDER/weather_dag.py', '--cfg-path', '/tmp/tmpzw3vc3u6']
[2023-09-15T11:31:18.513+0000] {standard_task_runner.py:85} INFO - Job 21: Subtask extract_data
[2023-09-15T11:31:18.616+0000] {task_command.py:410} INFO - Running <TaskInstance: weather_data.extract_data scheduled__2023-08-31T00:00:00+00:00 [running]> on host 4af8904e631c
[2023-09-15T11:31:18.824+0000] {taskinstance.py:1547} INFO - Exporting env vars: AIRFLOW_CTX_DAG_OWNER='***' AIRFLOW_CTX_DAG_ID='weather_data' AIRFLOW_CTX_TASK_ID='extract_data' AIRFLOW_CTX_EXECUTION_DATE='2023-08-31T00:00:00+00:00' AIRFLOW_CTX_TRY_NUMBER='1' AIRFLOW_CTX_DAG_RUN_ID='scheduled__2023-08-31T00:00:00+00:00'
[2023-09-15T11:31:19.127+0000] {python.py:183} INFO - Done. Returned value was: None
[2023-09-15T11:31:19.156+0000] {taskinstance.py:1350} INFO - Marking task as SUCCESS. dag_id=weather_data, task_id=extract_data, execution_date=20230831T000000, start_date=20230915T113118, end_date=20230915T113119
[2023-09-15T11:31:19.246+0000] {local_task_job_runner.py:225} INFO - Task exited with return code 0
[2023-09-15T11:31:19.362+0000] {taskinstance.py:2653} INFO - 0 downstream tasks scheduled from follow-on schedule check
